doc/src/conf.py

```diff
@@ -99,7 +99,6 @@
     "modules/physics/mechanics/symsystem": "../explanation/modules/physics/mechanics/symsystem.html",
     "modules/physics/mechanics/linearize": "../explanation/modules/physics/mechanics/linearize.html",
     "modules/physics/mechanics/sympy_mechanics_for_autolev_uses": "../explanation/modules/physics/mechanics/sympy_mechanics_for_autolev_uses.html",
-    "modules/physics/mechanics/examples": "../tutorials/physics/mechanics.html",
```

Comment:
Is this mistakenly removed? I would guess we want to keep this redirect.

Reply:
There are two items with the same key in this dict. The ruff check looks for duplicate keys.

Reply:
OK